calico_node/tests/st/bgp/test_ipip.py

```diff
@@ -90,47 +91,44 @@ def test_ipip(self, backend):
 
             # Turn on IPIP with a v1.0.2 calicoctl and check that the
             # IPIP tunnel is being used.
-            self.pool_action(host1, "replace", DEFAULT_IPV4_POOL_CIDR, True,
+            self.pool_action(host1, "replace", DEFAULT_IPV4_POOL_CIDR, ipip_mode="Always",
```

Comment:
same here, if we're testing older calicoctl then it should be `always` (lowercase)

Reply:
We're not.